eslintrc.js

```diff
@@ -71,10 +72,7 @@ module.exports = {
     "sort-requires/sort-requires": "error",
     "strict": ["error", "global"],
     "jest/no-alias-methods": "error",
-    "jest/prefer-to-be-null": "error",
-    "jest/prefer-to-be-undefined": "error",
-    "jest/prefer-to-contain": "error",
-    "jest/prefer-to-have-length": "error",
-    "jest/valid-expect-in-promise": "error",
+    "jest/no-disabled-tests": "error",
+    "jest/no-jasmine-globals": "error",
```

Comment:
The added 2 rules are both `warn` in `plugin:jest/recommended`, so I changed these to `error` level.